front/app/containers/Admin/projects/project/tabs.ts

```diff
@@ -53,7 +53,7 @@ export const getTabs = (
       feature: 'polls',
       name: 'poll',
     },
-    phase.attributes.participation_method === 'native_survey' && {
+    phase.attributes.supports_survey_form && {
       label: formatMessage(messages.surveyTab),
```

Comment:
We added a new phase attribute `supports_survey_form`, so we can replace a number of cases where we check if the `participation_method === 'native_survey'`. I'll create a separate PR for that cleanup though :)

Reply:
Yeah agreed - I'll let James know and we can remove it then :+1: